dev-python/pefile/pefile-9999.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2022 Gentoo Authors
```

Comment:
Can you try get tests running if possible?

See https://projects.gentoo.org/python/guide/distutils.html#enabling-tests.

(Always test with e.g. `ebuild ... clean test install` to run an ebuild's test suite.)

Reply:
Same for all other packages if they seem to have them, too.

Reply:
Upstream doesn't ship a test suite inside the release files. The test suite is only available in their Git repo and encrypted. It seems only their CI can decrypt the test files. Presumably they are trying to work around legal issues as the test suite contains file they are not allowed to redistribute. In short, there seems to be no way to test the package locally.

I didn't find any indication that there is a test suite in `leechcorepyc` and `volatility3`.

Reply:
Understood. Please mention this wihtin the ebuild so we don't have to wonder in future ;)